stream_buffer.c

```diff
@@ -497,11 +497,19 @@ size_t xStreamBufferSpacesAvailable( StreamBufferHandle_t xStreamBuffer )
 {
     const StreamBuffer_t * const pxStreamBuffer = xStreamBuffer;
     size_t xSpace;
+    volatile size_t xOriginalTail;

```

Comment:
Does this need to be `volatile`? It seems like the volatile `xTail` should be enough. I'd double-check this by running the test suite locally, bit it wasn't immediately obvious how to set that up. Is there a testing guide?

Reply:
I removed the volatile.